signin.js

```diff
@@ -52,39 +62,55 @@ passwordInput.addEventListener("keyup", function () {
 
 // test code
 const myEmail = "test@codeit.com";
-const myPassword = "code101";
+const myPassword = "sprint101";
 
 const logIn = document.querySelector(".sign-form");
 logIn.addEventListener("submit", function (e) {
   e.preventDefault();
+
   const email = emailInput.value;
   const password = passwordInput.value;
 
-  if (myEmail === email && myPassword === password) {
-    window.location.href = "./folder.html";
-  } else {
-    errorEmail.value = "이메일을 확인해 주세요.";
-    errorPassword.value = "비밀번호를 확인해 주세요.";
-  }
+  signIn(email, password);
 });
+// post 요청
+const signInUrl = "https://bootcamp-api.codeit.kr/api/sign-in";
+
+async function signIn(email, password) {
+  try {
+    const response = await fetch(signInUrl, {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+      },
+      body: JSON.stringify({
+        email: email,
+        password: password,
+      }),
+    });
+
+    if (response.status === 200) {
```

Comment:
🍯
fetch의 Promise fulfilled 객체 내부에는 `ok`라는 친구도 있답니다~
`response.ok` 속성은 HTTP 응답 상태 코드가 200~299 사이일 때 true를 반환해주기 때문에 이렇게 작성해도 돼요!

```suggestion
    if (response.ok) {
```